ast-model/src/org/jetbrains/dukat/astModel/LambdaParameterModel.kt

```diff
@@ -4,5 +4,6 @@ import org.jetbrains.dukat.astCommon.Entity
 
 data class LambdaParameterModel(
         override val name: String?,
-        override val type: TypeModel
+        override val type: TypeModel,
+        val hasType: Boolean
```

Comment:
let's name explicitelyDeclared or something like that - hasType is a bit confusing